window/src/lib.rs

```diff
@@ -264,6 +264,9 @@ pub trait WindowOps {
         Ok(())
     }
 
+    /// Hide other app windows
+    fn hide_other_apps(&self);
```

Comment:
This should have a default NOP implementation so that the build doesn't break for all the other systems

```suggestion
    fn hide_other_apps(&self) {}
```